api-reference/docstubs/api/calendar-list-calendarpermissions.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "List calendarPermissions"
+description: "Get the calendarPermissions from the calendarPermissions navigation property."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List calendarPermissions
+
+Namespace: microsoft.graph
+
+Get the calendarPermissions from the calendarPermissions navigation property.
```

Comment:
Shouldn't the description here be "Get a list of calendarPermissions"? calendarPermissions should be a link or have bold formatting applied.